end-to-end-test/shared/get_pullrequest_info.py

```diff
@@ -33,7 +33,7 @@
     frontend_group_id = "com.github."+frontend_user
 
     pr_state = ""
-    if jData['mergeable_state'] == 'draft':
+    if jData['draft'] == 'true':
```

Comment:
The problems seems to have been that github changed the way they reflect draft status in api